packages/design-system/src/components/Banner/Banner.js

```diff
@@ -0,0 +1,83 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import getMergedClassNames from '../../utils/getMergedClassNames';
+import CloseIcon from 'react-material-icon-svg/dist/CloseIcon';
+import AlertIcon from 'react-material-icon-svg/dist/AlertIcon';
+import InformationOutlineIcon from 'react-material-icon-svg/dist/InformationOutlineIcon';
+import BlockHelperIcon from 'react-material-icon-svg/dist/BlockHelperIcon';
+import CheckCircleIcon from 'react-material-icon-svg/dist/CheckCircleIcon';
+
+import styles from './style.scss';
+
+const cx = classNames.bind(styles);
+
+const baseClass = 'banner';
+
+export const Banner = props => {
+  const {
+    className,
+    closeIcon,
+    onClose,
+    size,
+    text,
+    type,
+    ...restProps
+  } = props;
+
+  const mergedWrapperClassNames = getMergedClassNames(
+    cx({
+      [`${baseClass}--close`]: true,
```

Comment:
please use BEM (block__element--modifier) here and in other class names:
```suggestion
      [`${baseClass}__close`]: true,
```

Reply:
I don't understand why you use i.e `${baseClass}--content--${size}` here and in the nested div. Something is wrong with this css